applications/luci-app-email/root/usr/share/rpcd/acl.d/luci-app-email.json

```diff
@@ -0,0 +1,24 @@
+{
+	"luci-app-email": {
+		"read": {
+			"ubus": {
+				"uci": [
+					"get"
+				]
+			},
```

Comment:
I think we do not need this explicit ubus acl for uci there. This acl is already granted within the next line.